.github/pull_request_template.md

```diff
@@ -0,0 +1,52 @@
+<!-- DO NOT REMOVE BEGIN -->
```

Comment:
Now that we already have a pull request template, should this overwrite the other one instead? (https://github.com/Unity-Technologies/com.unity.multiplayer.samples.bitesize/blob/main/.github/PULL_REQUEST_TEMPLATE.md)

Reply:
As you wish! The template I put here is just a proposal. @SamuelBellomo @fernando-cortez @jilfranco-unity  what are your opinions on this?

Reply:
+1 on being consistent with BR and adding bitesize specific stuff

Reply:
The one item that I see missing in the checklist is the callout to add a blurb to the CHANGELOG, if necessary. This bit here from the old template:

`Release notes have been added to the [project changelog](../CHANGELOG.md) file`

Reply:
Can we address this in a separate PR? The scope of this one is big enough, and I'd like this to get merged as soon as possible as it has been around for 3 months already

Reply:
I feel that it's important to bring it back in, since it's the only removal of what was already there.
It's a necessary step when creating PRs to attach a changelog addition, so if it's a one-liner we should bring it back.

Reply:
Good point, done!